src/layout/questionnaire-nav/components/arbo-simplifield.jsx

```diff
@@ -39,37 +39,38 @@ class ArboSimplified extends Component {
 
   renderComponentsByParent(components, parent) {
     const renderComponentsByParent = this.renderComponentsByParent;
-
     return getSortedChildren(components, parent).map(key => {
-      const subTree = renderComponentsByParent(components, key);
-      return (
-        <li
-          key={key}
-          className={isQuestion(components[key]) ? 'questions' : ''}
-        >
-          {components[key].children && components[key].children.length > 0 && (
-            <a
-              onClick={e => this.handleExpand(e, key)}
-              href="#"
-              aria-label="expand/collapse"
-            >
-              <span
-                className={`glyphicon ${
-                  this.state.expanded.indexOf(key) >= 0
-                    ? 'glyphicon-menu-down'
-                    : 'glyphicon-menu-right'
-                }`}
-              />
+      if(key !== "idendquest") {
+        const subTree = renderComponentsByParent(components, key);
+        return (
+          <li
+            key={key}
+            className={isQuestion(components[key]) ? 'questions' : ''}
+          >
+            {components[key].children && components[key].children.length > 0 && (
+              <a
+                onClick={e => this.handleExpand(e, key)}
```

Comment:
You should replace this element(a link with no href but with on onClick) with a button. It is semantically better.

Reply:
we did this already on other branch 'dev-loop' we wait the deployment of the 2 branches, and it will be fine